flow-typed/app.js

```diff
@@ -9,3 +9,80 @@ declare var process: { env: { PUBLIC_URL: string, NODE_ENV: string, REACT_APP_MO
   | 'phone_number'
   | 'relationship'
 
+type TranslateStringsType = {
```

Comment:
Ahh yeah I forgot about this. In the future we can use one language as the base for typing the rest and then we don't have to maintain this.